***На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9.***
---
Код содержит три цифры, которые нужно нажать одновременно. 
- Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?

1) Необходимо определить, сколько всего может быть комбинаций

In [6]:
from math import factorial

combinations = factorial(10) / (factorial(3) * factorial(10 - 3))

result = 1 / combinations
combinations, result

(120.0, 0.008333333333333333)

Ответ: вероятность того, что человек откроет дверь с первой попытки равна 1 / 120 или же 0.83%